src/screen/Council/CouncilScreen.tsx

```diff
@@ -2,19 +2,33 @@ import {useWindowDimensions} from 'react-native';
 import Identicon from '@polkadot/reactnative-identicon';
 import {createMaterialTopTabNavigator} from '@react-navigation/material-top-tabs';
 import {useNavigation} from '@react-navigation/native';
-import {Divider, Text} from '@ui-kitten/components';
+import {useAccounts} from 'context/AccountsContext';
+import {useApi} from 'context/ChainApiContext';
 import {EmptyView} from 'presentational/EmptyView';
 import LoadingView from 'presentational/LoadingView';
+import Padder from 'presentational/Padder';
 import SafeView, {noTopEdges} from 'presentational/SafeView';
-import React, {useMemo} from 'react';
+import {SelectAccount} from 'presentational/SelectAccount';
+import React, {useMemo, useState} from 'react';
 import {SectionList, StyleSheet, View} from 'react-native';
 import {useAccountIdentityInfo} from 'src/api/hooks/useAccountIdentityInfo';
 import {useCouncil} from 'src/api/hooks/useCouncil';
 import {useCouncilSummary} from 'src/api/hooks/useCouncilSummary';
+import {useFormatBalance} from 'src/api/hooks/useFormatBalance';
+import {getBalanceFromString} from 'src/api/utils/balance';
 import {candidateScreen} from 'src/navigation/routeKeys';
-import {Layout, List} from 'src/packages/base_components';
-import globalStyles, {standardPadding} from 'src/styles';
+import {Input, Text} from '@ui-kitten/components';
+import {List, Button, Divider, Modal, Card} from 'src/packages/base_components';
+import globalStyles, {monofontFamily, standardPadding} from 'src/styles';
 import {MotionsScreen} from './MotionsScreen';
+import {ScrollView, TouchableOpacity} from 'react-native';
+import {useModuleElections} from 'src/api/hooks/useModuleElections';
+import Badge from 'presentational/Badge';
+import {noop} from 'lodash';
+import {useApiTx} from 'src/api/hooks/useApiTx';
+import {useTheme} from 'context/ThemeContext';
+
+const MAX_VOTES = 16;
```

Comment:
I guess you got this value from the polkadotjs app, right? It seems like an arbitrary number. I wonder if we could get it from the api

Reply:
It is hard coded in the polkadot app as well.